src/pages/ImportData.js

```diff
@@ -119,9 +121,33 @@ const ImportData = () => {
       }],
       ['snapshot', async () => {
         snapshots && snapshots.length > 0 ?
-          await Promise.allSettled(_.map(({ title, id }) => Ajax().Workspaces.workspace(namespace, name).importSnapshot(id, title), snapshots)) :
-          await Ajax().Workspaces.workspace(namespace, name).importSnapshot(snapshotId, snapshotName)
-        notify('success', 'Snapshot imported successfully.', { timeout: 3000 })
+          await Promise.allSettled(
+            _.map(({ title, id, description }) => {
+              // Replace all whitespace characters with _
+              // Then replace all non alphanumeric characters with nothing
```

Comment:
This is what the code does, but my preference would be a comment here describing the why instead of the what.

Reply:
Updated!